toolkit/views/character_generator/character_generator.py

```diff
@@ -71,9 +71,17 @@ def post(self, request: HttpRequest):
             try:
                 if request.POST.get("generate_button") is not None:
 
-                    # TODO: get generator from page
-                    stat_generator_key = "random"
                     generator_key = "random"
+
+                    if form.generator_type.value == "3D6":
```

Comment:
This looks ridiculous now, but it allows the frontend names of the generator types to be different than the corresponding backend names. In the future, we may implement a new generator type and want show users a special name. This control block should support that design.